topics/contributing/tutorials/create-new-tutorial-content/tutorial.md

```diff
@@ -965,6 +965,63 @@ And in your text you can use braces to refer to the term
 > {: .code-out}
 {: .code-2col}
 
+## Choose Your Own Tutorial
```

Comment:
I think that it should be included in a different PR, since it doesn't correspond to the Ref-based RNA-seq tutorial.

Reply:
I put it into #3389 but not sure it was the best way to do it.